readme.md

```diff
@@ -139,8 +183,14 @@ Define a Javascript constructor called 'Instructor'. Every instance of Instructo
 Instantiate an instructor named 'Andy' and call its `receivePresent` method with "Resin Laying Deer Figurine, Gold" as the argument.
 
 ```js
-// Your code starts here...
-
+function Instructor(name) {
+  this.name = name;
+  this.receivePresent = function(gift) {
+    console.log(this.name +" promptly drops the "+ gift +" on the floor.");
+  }
+}
+var andy = new Instructor('Andy');
+andy.receivePresent("Resin Laying Deer Figurine, Gold");
 ```
 
```

Comment:
:+1:

Can also define `receivePresent` as a prototypal method since it is identical for every instance of an `Instructor`.